2021-09-26 04:42:57,543 - werkzeug - INFO - 1668 -  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
2021-09-26 04:42:57,549 - werkzeug - INFO - 1668 -  * Restarting with stat
2021-09-26 04:42:59,412 - werkzeug - WARNING - 1682 -  * Debugger is active!
2021-09-26 04:42:59,413 - werkzeug - INFO - 1682 -  * Debugger PIN: 139-850-208
2021-09-26 04:48:49,347 - werkzeug - INFO - 1682 -  * Detected change in '/mnt/c/Users/Mohit Khanwale/Desktop/SplitReq/batch_parser/flask_app/wsgi.py', reloading
2021-09-26 04:48:49,376 - werkzeug - INFO - 1668 -  * Restarting with stat
2021-09-26 04:48:51,220 - werkzeug - WARNING - 1747 -  * Debugger is active!
2021-09-26 04:48:51,220 - werkzeug - INFO - 1747 -  * Debugger PIN: 139-850-208
2021-09-26 04:50:42,825 - werkzeug - INFO - 1747 -  * Detected change in '/mnt/c/Users/Mohit Khanwale/Desktop/SplitReq/batch_parser/flask_app/flask_app_batch.py', reloading
2021-09-26 04:50:42,857 - werkzeug - INFO - 1668 -  * Restarting with stat
2021-09-26 04:51:46,921 - werkzeug - WARNING - 1821 -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-09-26 04:51:46,921 - werkzeug - INFO - 1821 -  * Running on http://172.23.14.40:5000/ (Press CTRL+C to quit)
2021-09-26 04:51:46,927 - werkzeug - INFO - 1821 -  * Restarting with stat
2021-09-26 04:51:48,723 - werkzeug - WARNING - 1835 -  * Debugger is active!
2021-09-26 04:51:48,724 - werkzeug - INFO - 1835 -  * Debugger PIN: 139-850-208
2021-09-26 04:54:37,820 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (0/20) now.
2021-09-26 04:54:37,821 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (1/20) in 1.00 second.
2021-09-26 04:54:38,822 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (2/20) in 1.00 second.
2021-09-26 04:54:39,824 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (3/20) in 1.00 second.
2021-09-26 04:54:40,826 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (4/20) in 1.00 second.
2021-09-26 04:54:41,828 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (5/20) in 1.00 second.
2021-09-26 04:54:42,829 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (6/20) in 1.00 second.
2021-09-26 04:54:43,831 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (7/20) in 1.00 second.
2021-09-26 04:54:44,832 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (8/20) in 1.00 second.
2021-09-26 04:54:45,834 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (9/20) in 1.00 second.
2021-09-26 04:54:46,836 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (10/20) in 1.00 second.
2021-09-26 04:54:47,837 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (11/20) in 1.00 second.
2021-09-26 04:54:48,839 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (12/20) in 1.00 second.
2021-09-26 04:54:49,841 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (13/20) in 1.00 second.
2021-09-26 04:54:50,843 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (14/20) in 1.00 second.
2021-09-26 04:54:51,844 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (15/20) in 1.00 second.
2021-09-26 04:54:52,846 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (16/20) in 1.00 second.
2021-09-26 04:54:53,848 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (17/20) in 1.00 second.
2021-09-26 04:54:54,272 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (0/20) now.
2021-09-26 04:54:54,273 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (1/20) in 1.00 second.
2021-09-26 04:54:54,849 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (18/20) in 1.00 second.
2021-09-26 04:54:55,274 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (2/20) in 1.00 second.
2021-09-26 04:54:55,851 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (19/20) in 1.00 second.
2021-09-26 04:54:56,276 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (3/20) in 1.00 second.
2021-09-26 04:54:56,853 - celery.backends.redis - CRITICAL - 1835 - 
Retry limit exceeded while trying to reconnect to the Celery redis result store backend. The Celery application must be restarted.

2021-09-26 04:54:57,278 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (4/20) in 1.00 second.
2021-09-26 04:54:57,353 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:54:57] "[35m[1mPOST /api/v1/cvparser/batch HTTP/1.1[0m" 500 -
2021-09-26 04:54:58,279 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (5/20) in 1.00 second.
2021-09-26 04:54:59,281 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (6/20) in 1.00 second.
2021-09-26 04:55:00,283 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (7/20) in 1.00 second.
2021-09-26 04:55:01,285 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (8/20) in 1.00 second.
2021-09-26 04:55:02,286 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (9/20) in 1.00 second.
2021-09-26 04:55:03,288 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (10/20) in 1.00 second.
2021-09-26 04:55:04,290 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (11/20) in 1.00 second.
2021-09-26 04:55:05,291 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (12/20) in 1.00 second.
2021-09-26 04:55:06,293 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (13/20) in 1.00 second.
2021-09-26 04:55:07,295 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (14/20) in 1.00 second.
2021-09-26 04:55:08,296 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (15/20) in 1.00 second.
2021-09-26 04:55:09,298 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (16/20) in 1.00 second.
2021-09-26 04:55:10,300 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (17/20) in 1.00 second.
2021-09-26 04:55:11,301 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (18/20) in 1.00 second.
2021-09-26 04:55:12,303 - celery.backends.redis - ERROR - 1835 - Connection to Redis lost: Retry (19/20) in 1.00 second.
2021-09-26 04:55:13,305 - celery.backends.redis - CRITICAL - 1835 - 
Retry limit exceeded while trying to reconnect to the Celery redis result store backend. The Celery application must be restarted.

2021-09-26 04:55:13,708 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:55:13] "[35m[1mPOST /api/v1/cvparser/batch HTTP/1.1[0m" 500 -
2021-09-26 04:56:06,564 - flask_app_batch - INFO - 1835 - <celery.backends.redis.RedisBackend object at 0x7f0004933e80>
2021-09-26 04:56:06,565 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:06] "POST /api/v1/cvparser/batch HTTP/1.1" 200 -
2021-09-26 04:56:06,587 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:06] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:06,596 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:06] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:09,606 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:09] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:12,616 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:12] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:15,638 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:15] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:18,649 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:18] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:21,661 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:21] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:24,670 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:24] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:27,690 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:27] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 04:56:30,710 - werkzeug - INFO - 1835 - 127.0.0.1 - - [26/Sep/2021 04:56:30] "GET /api/v1/cvparser/batch/task_result/509198c8-a854-4ca5-a4e1-fcea3d9c5e50 HTTP/1.1" 200 -
2021-09-26 05:00:44,931 - werkzeug - WARNING - 4836 -  * Running on all addresses.
   WARNING: This is a development server. Do not use it in a production deployment.
2021-09-26 05:00:44,932 - werkzeug - INFO - 4836 -  * Running on http://172.23.14.40:5000/ (Press CTRL+C to quit)
